eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Snapshots.hs

```diff
@@ -50,6 +51,9 @@ data ProposalsSnapshot era = ProposalsSnapshot
   }
   deriving (Generic, Eq, Show)
 
+proposalsSnapShotGovActionStatesL :: Lens' (ProposalsSnapshot era) (Map (GovActionId (EraCrypto era)) (GovActionState era))
+proposalsSnapShotGovActionStatesL = lens psGovActionStates $ \x y -> x {psGovActionStates = y}
```

Comment:
This lens violates the invariant that is hidden by this Snapshots interface. It should be removed.

Reply:
I'm removing it then :+1: